performance/src/main/java/org/hibernate/validator/performance/simple/SimpleValidation.java

```diff
@@ -71,13 +56,12 @@
 	@Warmup(iterations = 10)
 	@Measurement(iterations = 20)
 	public void testSimpleBeanValidation(ValidationState state, Blackhole bh) {
-		DriverSetup driverSetup = new DriverSetup( state );
-		Set<ConstraintViolation<Driver>> violations = state.validator.validate( driverSetup.getDriver() );
-		assertThat( violations ).hasSize( driverSetup.getExpectedViolationCount() );
+		Set<ConstraintViolation<Driver>> violations = state.validator.validate( state.driver );
+		assertThat( violations ).hasSize( 3 );
 		bh.consume( violations );
```

Comment:
I thought about that and I wonder if we should have 2 different benchmarks: one without violation and one with.

Why? Because most of the time, we expect HV to not return any violation so getting this path as fast as possible is definitely the priority. Whereas having the violation path a bit slower is not that big a deal.

Does it make sense?

Reply:
yes it does. You are right - users expect their models to be valid. 
+1 for two benchmarks.